go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	emperror.dev/errors v0.8.1
 	github.com/MakeNowJust/heredoc v1.0.0
 	github.com/andreyvit/diff v0.0.0-20170406064948-c7f18ee00883
-	github.com/cisco-open/operator-tools v0.30.0
```

Comment:
This change seems a bit suspicious, was it accidentally pushed?

Reply:
no, I forgot to mark this as a draft, we were waiting for a reviewer with write access here: https://github.com/cisco-open/operator-tools/pull/181